recipes/flyteidl/LICENSE

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
This is already included in the tar ball.

Reply:
Gotcha. Removed the file.